h/static/scripts/app.coffee

```diff
@@ -7,6 +7,9 @@ require('angular-jwt')
 streamer = require('./streamer')
 
 resolve =
+  # Ensure that we have feature flags available before we load the main
+  # view as features such as groups affect which annotations are loaded
+  features: ['features', (features) -> features.fetch()]
```

Comment:
This is a funny one. The object you pass to `resolve` is [supposed to provide additional injectable objects](https://code.angularjs.org/1.4.7/docs/api/ngRoute/provider/$routeProvider)... so I would expect what you return from this function to overwrite the `features` service.

Clearly this doesn't happen, otherwise you'd have noticed (and indeed I've just verified that `'features'` still injects the features service) but perhaps to be on the safe side we could call this `featuresLoaded` or something.